apparmor.d/profiles-g-l/lsd

```diff
@@ -0,0 +1,22 @@
+# apparmor.d - Full set of apparmor profiles
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{,usr/}{,local/}bin/lsd
+profile lsd @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/nameservice-strict>
+  include <abstractions/consoles>
+
+  capability dac_read_search,
+
+  @{exec_path} mr,
+
+  / r,
+  /**/ r,
```

Comment:
Thoughts? Third-party, sure we don't want to spin up a sandbox for directory traverse.